Servlets/src/main/java/ua/goit/servlets/model/Project.java

```diff
@@ -0,0 +1,30 @@
+package ua.goit.servlets.model;
+
+public class Project {
+  final private Integer id;
+  final private String name;
+  private String description;
```

Comment:
let description also be private to be consistent with the rest. also usually people put it like `private final`